script.js

```diff
@@ -0,0 +1,23 @@
+var fName = document.getElementById("fName");
+var lName = document.getElementById("lName");
+var full = document.getElementById("full");
+trimmed2 = "";
+function lengthCheck() {
+    if (fName.value.length > 20 || lName.value.length > 20 ) {
+        full.value = "";
+        return 0;
+      }
+    
+}
+fName.addEventListener ('input',function() {
```

Comment:
It should be written like this ->
`fName.addEventListener('input', function() {`
Follow this standard.